fastpass/src/main/scala/scala/meta/internal/fastpass/bazelbuild/BloopBazel.scala

```diff
@@ -136,7 +136,7 @@ object BloopBazel {
                 Bazel.PlatformClasspathLabel ::
                 Bazel.ProtobufJava ::
                 importedTargets.map(_.getRule.getName)
-            actions <- bazel.aquery(actionGraphLabels)
+            actions <- bazel.aquery(actionGraphLabels, 0)
```

Comment:
Instead of passing the number of the current try here, why not pass how many times the call may be retried? In `aquery` you can then do re-tries until the count reaches 0.